organizations/info.yml

```diff
@@ -2,12 +2,10 @@
 query:
   model: true
   sql: |
-    select *
-    from decidim_organizations
-    where host = HOST
+    select * from decidim_organizations where host = $HOST
```

Comment:
Keep multilines queries as it allows more readability, keep $HOST as it's correct.